pymeasure/instruments/kepco/kepcoklp75.py

```diff
@@ -0,0 +1,163 @@
+            name=name,
+            read_termination="\n",
+            write_termination="\n",
+            **kwargs
+        )
+
+    output_enabled = Instrument.control(
+        "OUTPut:STATe?", "OUTPut:STATe %d",
+        "Control whether the output is enabled or disabled.",
+        validator=strict_discrete_set,
+        values={True: 1, False: 0},
+        map_values=True,
+    )
+
+    voltage = Instrument.measurement(
+        "MEASure:VOLTage?",
+        "Measure the voltage across the output terminals (V).",
+        cast=float,
+    )
+
+    current = Instrument.measurement(
+        "MEASure:CURRent?",
+        "Measure the current through the output terminals (A).",
+        cast=float,
+    )
+
+    voltage_setpoint = Instrument.control(
+        "SOURce:VOLTage?", "SOURce:VOLTage %.1f",
+        "Set or get the voltage setpoint (0-75 V).",
+        validator=truncated_range,
+        values=[0, _Vmax],
+    )
+
+    current_setpoint = Instrument.control(
+        "SOURce:CURRent?", "SOURce:CURRent %.1f",
+        "Set or get the current setpoint (0.4-16 A).",
+        validator=truncated_range,
+        values=[_Imin, _Imax],
+    )
+
+    def enable_protection(self, ovp=None, ocp=None):
+        """
+        Control the enabling of overvoltage or overcurrent protection.
+
+        Parameters:
+            ovp (float, optional): Overvoltage protection threshold in volts (V).
+            ocp (float, optional): Overcurrent protection threshold in amperes (A).
+
+        Raises:
+            ValueError: If the specified thresholds are out of valid range.
+        """
+        if ovp is not None:
+            if ovp < 0 or ovp > self._Vmax:
+                raise ValueError(f"OVP value {ovp} is out of range (0-{self._Vmax} V).")
+            self.write(f"SOURce:VOLTage:PROTection {ovp}")
+        if ocp is not None:
+            if ocp < self._Imin or ocp > self._Imax:
+                raise ValueError(f"OCP value {ocp} is out of range ({self._Imin}-{self._Imax} A).")
+            self.write(f"SOURce:CURRent:PROTection {ocp}")
+
+    def status(self):
+        """
+        Query the instrument's status.
+
+        Returns:
+            str: The status message returned by the instrument.
+        """
+        return self.ask("STATus:QUEStionable?")
+
+    confidence_test = Instrument.measurement(
+        "*TST?",
+        "Measure the result of the self-test and return the error code.",
+        cast=int,
+    )
+
+    def reset(self):
+        """
+        Control the reset of the instrument to its default state.
+        """
+        self.write("*RST")
+
+    def clear(self):
+        """
+        Control the clearing of the instrument's error queue.
+        """
+        self.write("*CLS")
+
+    def get_id(self):
+        """
+        Get the instrument's identification string.
+
+        Returns:
+            str: The identification string of the instrument.
+        """
+        return self.ask("*IDN?")
```

Comment:
These commands should be already defined in SCPIMixin (check it), so they are superfluous here.